arm-sql/2015-05-01-preview/swagger/virtualNetworkRules.json

```diff
@@ -0,0 +1,295 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Virtual Network Rules API provides a RESTful set of web APIs that interact with firewall rules to manage your virtual networks. The API enables users to create, retrieve, update, and delete virtual network rules."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/virtualNetworkRules/{vnetFirewallRuleName}": {
+      "get": {
+        "tags": [
+          "VnetFirewallRule"
+        ],
+        "description": "Returns information about a Vnet Firewall Rule.",
```

Comment:
Ill change it to virtual network rules since thats what we are calling it to the customers.